### PR TITLE
Fix #3589: Deleted phone landscape xml and tablet potrait xml.

### LINKED ISSUE #3589: Merge topic_summary_view.xml into single xml file
Currently there are 3 versions of `topic_summary_view.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by ishant904**
Fixes #3589 
Deleted phone landscape xml and tablet potrait xml.

## Essential Checklist
- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

| Old SS                                                                                                                          | New SS                                                                                                                          |
|---------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
| Screenshot_1639417989 | Screenshot_1642015927 |
| Screenshot_1639418725 | Screenshot_1642015974 |
| Screenshot_1639419482 | Screenshot_1642016272 |
| Screenshot_1639419467 | Screenshot_1642017139 |

The first two are phone SS and last two are tablet SS

### CODE REVIEW

**Comment by yash10019coder** [comment]
Hi @ishant904 it is still incorrect take a look at this PR #3879

**Comment by ishant904** [comment]
@yash10019coder i have made some changes, let me know if anything else needs to be changed

**Comment by ishant904** [comment]
@Rigzbot sorry I have updated the PR

**Comment by yash10019coder** [comment]
> Hi @ishant904 it is still incorrect take a look at this PR #3879

@ishant904 still incorrect take a look at this pr

**Comment by ishant904** [comment]
@yash10019coder Please correct me if I am wrong, #3589 is the issue and the PR #3879 which you mentioned is referring to #3859 in the title. If you can please lead me to the particular information which needs to be changed.

**Comment by yash10019coder** [comment]
> @yash10019coder Please correct me if I am wrong, #3589 is the issue and the PR #3879 which you mentioned is referring to #3859 in the title. If you can please lead me to the particular information which needs to be changed.

your work is correct you just need to display the screenshots in table like format.
like done in this PR #3879

**Comment by yash10019coder** [comment]
[You can view this blog post for reference](https://www.markdownguide.org/extended-syntax/)

**Comment by ishant904** [comment]
@yash10019coder thanks for pointing that out. I have made some changes. Let me know if any changes are required

**Comment by ishant904** [comment]
@yash10019coder I was out for couple of weeks and now the PR is closed. Any way to reopen it or should I create a new one?

**Comment by yash10019coder** [comment]
No problem @ishant904  opened the PR

**Comment by ishant904** [comment]
thanks @yash10019coder . I have resolved conflict, please do review the PR

File: app/src/main/res/values/dimens.xml [hunk 1]
```diff
@@ -588,9 +588,13 @@
...
 
+  <!--Topic Summary-->
+  <dimen name="topic_summary_lesson_count_text_view_padding_bottom">8dp</dimen>
+  
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
    
   <!-- StoryFragment -->
   <dimen name="story_fragment_padding_bottom">160dp</dimen>
+
```
Comment (yash10019coder) [inline app/src/main/res/values/dimens.xml:615]: unnecessary new line

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 2]
```diff
@@ -396,7 +396,10 @@
   <dimen name="section_title_text_view_layout_margin_top">32dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">96dp</dimen>
 
+  <!--Topic Summary-->
+  <dimen name="topic_summary_lesson_count_text_view_padding_bottom">12dp</dimen>
+  
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
-
+
```
Comment (yash10019coder) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: same goes here

**Review by yash10019coder — COMMENTED** [review]
@ishant904  reformat the code and also try to remove swp600 portrait

**Comment by yash10019coder** [comment]
and @rt4914  can you please approve the workflow

**Comment by ishant904** [comment]
@yash10019coder removed sw600dp layout. Let me know on additional changes

**Comment by yash10019coder** [comment]
> @yash10019coder removed sw600dp layout. Let me know on additional changes

@ishant904  is the app building now?

**Comment by ishant904** [comment]
@yash10019coder yes, I have updated the SS

**Comment by ishant904** [comment]
@yash10019coder please do review the changes and let me know if any other changes are required. @rt4914 please do approve the workflow.

File: app/src/main/res/layout/topic_summary_view.xml [hunk 3]
```diff
@@ -66,7 +67,6 @@
           android:textColor="@color/white"
           android:textSize="14sp"
           android:textStyle="bold"
-          app:layout_constraintHorizontal_bias="0"
```
Comment (rt4914) [inline app/src/main/res/layout/topic_summary_view.xml]: Keep is attribute.

File: app/src/main/res/layout/topic_summary_view.xml [hunk 4]
```diff
@@ -31,6 +31,7 @@
         android:layout_width="0dp"
         android:layout_height="0dp"
         android:importantForAccessibility="no"
+        android:scaleType="centerInside"
```
Comment (rt4914) [inline app/src/main/res/layout/topic_summary_view.xml]: Remove this as it is not required.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ishant904 Please fix suggestions mentioned by me and @yash10019coder . Also reply to all the conversations.

**Comment by rt4914** [comment]
@ishant904 Please merge with latest develop.

**Comment by ishant904** [comment]
@rt4914 i have made the changes as suggested by you and @yash10019coder.

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 2 above)
Comment (ishant904) [reply app/src/main/res/values-sw600dp-land/dimens.xml]: removed newlines

File: app/src/main/res/values/dimens.xml (hunk 1 above)
Comment (ishant904) [reply app/src/main/res/values/dimens.xml:615]: removed newlines

File: app/src/main/res/layout/topic_summary_view.xml (hunk 3 above)
Comment (ishant904) [reply app/src/main/res/layout/topic_summary_view.xml]: Done

File: app/src/main/res/layout/topic_summary_view.xml (hunk 4 above)
Comment (ishant904) [reply app/src/main/res/layout/topic_summary_view.xml]: Done

**Comment by ishant904** [comment]
@FareesHussain i see couple of test failed. Let me know the further actions. Also I unassigned myself by mistake.

File: app/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -586,6 +586,9 @@
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
 
+  <!--Topic Summary-->
```
Comment (rt4914) [inline app/src/main/res/values/dimens.xml]: Re-write as 
`<!-- TopicSummaryView -->`
One space before and after text

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 6]
```diff
@@ -396,6 +396,9 @@
   <dimen name="section_title_text_view_layout_margin_top">32dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">96dp</dimen>
 
+  <!--Topic Summary-->
```
Comment (rt4914) [inline app/src/main/res/values-sw600dp-land/dimens.xml]: `<!-- TopicSummaryView -->`

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ishant904 PTAL thanks.

**Comment by FareesHussain** [comment]
> @FareesHussain i see couple of test failed. Let me know the further actions. Also I unassigned myself by mistake.

Try to read the failure checks, look for the key words `failed` `error` and look for the reason why it fails. Let me know if you are unable to figure out the issue

File: app/src/main/res/values/dimens.xml (hunk 5 above)
Comment (ishant904) [reply app/src/main/res/values/dimens.xml]: Done

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 6 above)
Comment (ishant904) [reply app/src/main/res/values-sw600dp-land/dimens.xml]: Done

**Comment by ishant904** [comment]
@rt4914 @yash10019coder @FareesHussain Please do review it.

File: app/src/main/res/values-sw600dp-land/dimens.xml [hunk 7]
```diff
@@ -410,5 +413,4 @@
 
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
-
```
Comment (yash10019coder) [inline app/src/main/res/values-sw600dp-land/dimens.xml:413]: please remove this newline as it is unnecessary changes

File: app/src/main/res/values/dimens.xml [hunk 8]
```diff
@@ -606,7 +609,7 @@
 
   <!-- AudioLanguageFragment -->
   <dimen name="audio_language_recycler_view_padding_bottom">132dp</dimen>
-
```
Comment (yash10019coder) [inline app/src/main/res/values/dimens.xml:609]: this too

**Review by yash10019coder — CHANGES_REQUESTED** [review]
@ishant904 I have mentioned some changes PTAL thanks

**Comment by yash10019coder** [comment]
and @rt4914 please approve the workflows

File: app/src/main/res/values-sw600dp-land/dimens.xml (hunk 7 above)
Comment (ishant904) [reply app/src/main/res/values-sw600dp-land/dimens.xml:413]: i think all the extra newlines are removed. can you point out the line number.

**Comment by ishant904** [comment]
@FareesHussain @rt4914 @yash10019coder I looked into one of the failed CI checks and it says that gradle daemon disappered unexpectedly. Any suggestion?

**Comment by FareesHussain** [comment]
re-ran the jobs lets see if it passes

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by ishant904** [comment]
@yash10019coder @rt4914 @FareesHussain one of the CI test is getting failed repeatedly. Looking into error msg, it states the gradle daemon was killed/crashed while running the build. I think it maybe because of low memeory allocated. Let me know your thoughts.

**Review by yash10019coder — APPROVED** [review]
LGTM @ishant904 Sorry for the delay, I think that it's a flaky test I'll rerun the ci checks.

**Comment by ishant904** [comment]
Test cases passed. Thanks @yash10019coder
